sources/packages/backend/apps/api/src/route-controllers/index.ts

```diff
@@ -55,5 +55,6 @@ export * from "./confirmation-of-enrollment/confirmation-of-enrollment.controlle
 export * from "./confirmation-of-enrollment/confirmation-of-enrollment.aest.controller";
 export * from "./overaward/overaward.controller.service";
 export * from "./overaward/overaward.aest.controller";
+export * from "./overaward/overaward.institution.controller";
```

Comment:
file name should also be plural, https://github.com/bcgov/SIMS/pull/1906/files#r1178245716